packages/apollo-client/src/error-helpers.ts

```diff
@@ -0,0 +1,19 @@
+export function getErrorString(error: any): string | undefined {
+    if (error) {
+        if (Array.isArray(error)) {
+            let s = "";
+            for (let i of (error as [])) {
+                if (s) s += "\n";
+                s += getErrorString(i);
+            }
+            return s;
+        }
+
+        if (typeof error !== 'string') {
+            error = error.message;
```

Comment:
This is usually what the error object looks like. I figure just use the "message" property:

```js
{
  "message": "404 Not Found",
  "locations": [
    {
      "line": 2,
      "column": 3
    }
  ],
  "path": [
    "product"
  ],
  "extensions": {
    "code": "INTERNAL_SERVER_ERROR",
    "exception": {
      "response": {
        "size": 0,
        "timeout": 0
      }
    }
  }
}
```